frontend/packages/core/src/card.tsx

```diff
@@ -51,23 +49,65 @@ const StyledCardHeaderContainer = styled.div({
   background: "#EBEDFB",
 });
 
-interface CardHeaderProps extends Pick<MuiCardHeaderProps, "avatar" | "title"> {
+const StyledCardHeader = styled(Grid)({
+  padding: "16px",
+  minHeight: "72px",
+  margin: "0px",
+  width: "100%",
+});
+
+const StyledCardHeaderAvater = styled.div({
+  marginRight: "16px",
+  fontSize: "24px",
+});
+
+const StyledDivider = styled(Divider)({
+  color: "rgba(13, 16, 48, 0.38)",
+  height: "36px",
+  alignSelf: "center",
```

Comment:
```suggestion
  alignSelf: "stretch",
```

Reply:
with `stretch`,  this happens when the screen size gets smaller:
<img width="392" alt="Screen Shot 2021-09-01 at 2 10 38 PM" src="https://user-images.githubusercontent.com/39421794/131763245-1bc77383-46cf-4f2e-9b55-268c21374f64.png">